src/api.py

```diff
@@ -131,7 +131,10 @@ def post_xml():
 
     # generate
     try:
-        article_json = scraper.main(path, {'override': override})
+        article_json = scraper.main(path, {
+            'override': override,
+            'fill-missing-image-dimensions': False
```

Comment:
shouldn't this be passed as `True` in this `api` scenario then?

Reply:
now that we only need an XML we should be able to call the API with a fixture (XML with figures) and validate the result?

Reply:
> shouldn't this be passed as True in this api scenario then?
nuts, typo.

> now that we only need an XML we should be able to call the API with a fixture (XML with figures) and validate the result?

I'll ping you in the Jira ticket, the XML has been uploaded there. Add it to the continuum tests?

Reply:
> Add it to the continuum tests?

or I could add it to the regular test suite...